.evergreen/csfle/stop_servers.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Unlike other executable files in this folder, this one doesn't have the executable bit set.

Reply:
Fixed